sdk/core/azure-core-tracing-opencensus/src/test/java/com/azure/core/tracing/opencensus/OpenCensusTracerTest.java

```diff
@@ -199,8 +199,6 @@ public void startSpanOverloadNullPointerException() {
         assertThrows(NullPointerException.class, () -> openCensusTracer.start("", Context.NONE, null));
     }
 
```

Comment:
There are no tests added?

Reply:
I think that was a leftover comment and addLinks was already tested in OpenCensus.